current/image-analysis.md

```diff
@@ -0,0 +1,48 @@
+## Image Analysis
+
+AUT supports image analysis, a growing area of interest within web archives.
```

Comment:
I've been trying to move us away from acronyms all over the place, so maybe just `The Toolkit`